docs/support/FAQ.md

```diff
@@ -192,7 +192,7 @@ If you need more control over the timing of releases, see [Triggering a release]
 
 ## Can I set the initial release version of my package to `0.0.1`?
 
-This is not supported by **semantic-release** as it's not considered a good practice, mostly because [Semantic Versioning](https://semver.org) rules applies differently to major version zero.
+This is not supported by semantic-release. [Semantic Versioning](https://semver.org/) rules apply differently to major version zero and supporting those differences is out of scope.
```

Comment:
```suggestion
This is not supported by semantic-release. [Semantic Versioning](https://semver.org/) rules apply differently to major version zero and supporting those differences is out of scope and not one of the goals of the semantic-release project.
```

@travi following https://github.com/semantic-release/semantic-release/pull/2993#issuecomment-1743837952, what do you think of this addition?

Reply:
That works for me. Thanks for humoring my long winded justification :)

Reply:
I like it 👍🏼

Reply:
Yay! I like having the goals clarification a lot, +1 that without it it's kind of unclear _why_ pre-0 is out of scope.

(sorry for responding to three paragraphs with a rushed answer - but yes agreed all around, and I really appreciate the write-up!)